src/CMakeLists.txt

```diff
@@ -740,14 +744,12 @@ else()
 		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DWIN32_LEAN_AND_MEAN")
 	endif()
 
-	set(CMAKE_CXX_FLAGS_RELEASE "-DNDEBUG ${RELEASE_WARNING_FLAGS} ${WARNING_FLAGS} ${OTHER_FLAGS} -Wall -pipe -funroll-loops")
```

Comment:
Please keep these flags for Linux if *BSD doesn't know them.

Reply:
It actually knows them, but they are broken.

Reply:
@SmallJoker I'm sorry for my ignorance, they aren't broken, they are just set by user or ports.